### PR TITLE
AP_AHRS: clarify frame of get_location_from_origin_offset

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

**Review by IamPete1 — COMMENTED** [review]
I'm not sure we really need these helpers. Looks like one is not used at all and the other is used in one place. 

I tend to think we should just remove them both....

File: libraries/AP_AHRS/AP_AHRS.cpp [hunk 1]
```diff
@@ -3536,7 +3536,7 @@ bool AP_AHRS::get_velocity_NED(Vector3f &vec) const
 
 // return location corresponding to vector relative to the
```
[the remark this answers is not in the document]
Comment (IamPete1) [reply libraries/AP_AHRS/AP_AHRS.cpp:3537]: Or at least confusing.

**Comment by peterbarker** [comment]
> I'm not sure we really need these helpers. Looks like one is not used at all and the other is used in one place.
> 
> I tend to think we should just remove them both....

Well, I was just adding a new user :-)

These can replace the constructor `Location(Vector3f, AltFrame)` - which can't actually tell you that origin isn't set now and will give you something very close to 0,0....

File: libraries/AP_AHRS/AP_AHRS.cpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterbarker) [reply libraries/AP_AHRS/AP_AHRS.cpp:3537]: Suggestions for less-confusing words welcome :-)

**Review by tridge — APPROVED** [review]

**Pull request merged**